src/components/Pomodoro/CircularProgress.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { StyledSvg, StyledCircle } from './styles/CircularProgress.styled'
+
+const CircularProgress = ({ percent }: { percent: number }) => {
+  const getPercentage = () => {
+    return (1 - percent) * 1055
+  }
+
+  return (
+    <StyledSvg transform="rotate(-90)">
```

Comment:
can consider placing the transform attribute in styled.ts? it seems to work fine